README.md

```diff
@@ -1 +1,16 @@
-# CS438_Project
\ No newline at end of file
+# CS438_Project
+
+
+#### Build project
+`go build`
+
+#### Build CLI client
+1. `cd client`
+2. `go build`
+
+
+#### How to start GUI:
+1. `cd web-react`   
+2. `npm install`
+3. `./CS438_Project`
```

Comment:
why this line?
isn't it just `cd web-react && npm install && npm start`?

Reply:
Yes, you can start the GUI without it. 
You should also start the Gossiper if you want your GUI to be connected to it.